conflictmonitor/mongo/mongo_initialize.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This script seems to be all commented out. Maybe we can remove this if it isn't used?

Reply:
removed